js/main.js

```diff
@@ -29,6 +29,13 @@ var TYPE_OF_HOUSES = [
   'bungalo'
 ];
 
+var HOUSES_NAMES = {
```

Comment:
```suggestion
var HousesNames = {
```

> var StatusCode = {
> OK: 200,
> NOT_FOUND: 404,
> BAD_REQUEST: 400,
> };

у меня, у предыдущей студентки были замечания по правильному написанию перечислений. Так что надо писать прям тютелька в тютельку как в критериях. На работе я пишу более свободно, в основном кемел кейсом. Хотя надо попробовать как-нибудь через константную запись, как тут в Академии :)